journal/vendor-extra/Views/ViewConverter.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Libero\Views;
+
+use Libero\Dom\Node;
+
+interface ViewConverter
+{
+    public function convert(Node $object, ?string $template, array $context = []) : ?View;
```

Comment:
Better to use `Element` rather than `Node`, but means handling text nodes separately everywhere.

Reply:
Now thinking this should be split up: have separate block (which can just use `Element`s, and require a template), and inline (which either use text as well or handle them different, and have no set template) converters.